docs/workflow.md

```diff
@@ -109,7 +110,7 @@ The Pull Request process usually looks like the following:
     1. **If you are a reviewer:**
        1. Look over the code, keeping an eye out for typos or bugs
```

Comment:
Can we link the naming conventions doc here somewhere?

Reply:
good idea